contracts/ChildChainGaugeInjectoooor.sol

```diff
@@ -345,18 +362,18 @@ contract ChildChainGaugeInjector is ConfirmedOwner, Pausable, KeeperCompatibleIn
      * @notice Sets the address of the ERC20 token this contract should handle
    */
     function setInjectTokenAddress(address ERC20token) public onlyOwner {
```

Comment:
Is there any scenario in which you actually change the token address? If not, I would recommend removing this and just making `s_injectTokenAddress` immutable

Reply:
why was this marked as resolved?